### ISSUE TITLE
Suggestion: More Rule Options in Multiplayer 

### ISSUE THREAD

**Issue by realPhint** [Sh0wdown/UnofficialCrusaderPatch#594]
There are alredy options for "no dogs" and "no cows" in SC Multiplayer. Some other common multiplayer-rules are no fire and no flying stones (wich I dont like). But anyway maybe those two options could be added, too. Meaning that by chosing option "no fire" all possible causes of fire (slaves, fireballista, fire throwers, etc.) become inaccassable in the game, while by chosing "no flying stones" all entities throwing stones (catapult, etc.) would become inaccessible (preventing mass catapults).

Another option would be to set a limit to like a maximum of 10 catapults or so.

again thanks for the great work! I'm still fascinated by this! more suggestions are coming ;)

**Comment by GRhin** [comment]
Just wanted to point out that no fire games allow slaves for moat digging,
and no flying stones allow catapults and trebuchets for throwing cows.

**Comment by GRhin** [comment]
Further, as I understand it, maps could be modified to disallow these units currently

**Comment by realPhint** [comment]
Thanks GRhin for the additional information.
I didn't know if it's possible to implement it that way, meaning slaves (or other units using fire) become unable to set buildings on fire at all and catapults etc. are simply unable to throw stones.

I guess having it as an option would be much more flexible than modifying maps :)

**Comment by GRhin** [comment]
Nah, I doubt that us possible, not for slaves at the least, for cats and
trebs might be able to force available stones to stay at 0 or something.
But the slaves are the more important one for games that use these
settings, digging a moat is slow and expensive without them. Maybe dropping
price of spears to compensate would be good, or removing the weapon
requirement.

**Comment by Krarilotus** [comment]
Another common rule is no disbanding of units for gold
Noufog or sth like that usually refered to in GameRanger games. I would say this is a neat rule and would require to diasable the disband feature.

**Comment by GRhin** [comment]
Or hijack the disband feature to kill units instead.

**Comment by realPhint** [comment]
Interesting ideas :)

**Comment by Luke-Mason** [comment]
@GRhin Can you add max catapults & trbuchets? so after 5, you can't make anymore? I'd love that for stopping people massing catapults, and improving strategy late game

**Comment by GRhin** [comment]
I can't do anything really. Maybe a dev would look into it. However for most Devs rebalancing the units would be a more attractive solution, and is within current abilities. 
I believe krarilotus is currently running a multiplayer mode using a unit rebalance that typically does not encounter mass cat spam, and definitely improving strategy throughout the entire game. @Krarilotus is this still being run via your YouTube channel?

**Comment by Krarilotus** [comment]
I haven't put up any youtube Videos/streams lately but you can join our discord community:
https://discord.gg/mU8RxYZ75E
(games are more commonly hosted during he weekend but also during the week in the 'unholy hours')
-> unfortunately i am very busy with UCP3.0 preparations so wont take part much of anything lately, as i want to progress and make further balance changes possible that we can't do with the current framework. And besides: It is really time that we get UCP3.0 out the door!
